src/main/java/com/stripe/model/Sku.java

```diff
@@ -15,140 +16,221 @@
+
+  @SerializedName("updated")
   Long updated;
-  Boolean deleted;
 
-  // <editor-fold desc="product">
+  /** Get id of expandable `product` object. */
   public String getProduct() {
     return (this.product != null) ? this.product.getId() : null;
   }
 
-  public void setProduct(String productId) {
-    this.product = setExpandableFieldId(productId, this.product);
-
+  public void setProduct(String id) {
+    this.product = ApiResource.setExpandableFieldId(id, this.product);
   }
 
+  /** Get expanded `product`. */
   public Product getProductObject() {
     return (this.product != null) ? this.product.getExpanded() : null;
   }
 
-  public void setProductObject(Product product) {
-    this.product = new ExpandableField<>(product.getId(), product);
-  }
-  // </editor-fold>
-
-  // <editor-fold desc="create">
-  /**
-   * Create a SKU.
-   */
-  public static Sku create(Map<String, Object> params) throws StripeException {
-    return create(params, null);
+  public void setProductObject(Product expandableObject) {
+    this.product = new ExpandableField<Product>(expandableObject.getId(), expandableObject);
   }
 
   /**
-   * Create a SKU.
+   * Retrieves the details of an existing SKU. Supply the unique SKU identifier from either a SKU
+   * creation request or from the product, and Stripe will return the corresponding SKU information.
    */
-  public static Sku create(Map<String, Object> params, RequestOptions options)
-      throws StripeException {
-    return request(RequestMethod.POST, classUrl(Sku.class), params, Sku.class, options);
+  public static Sku retrieve(String id) throws StripeException {
+    return retrieve(id, (Map<String, Object>) null, (RequestOptions) null);
   }
-  // </editor-fold>
 
-  // <editor-fold desc="delete">
   /**
-   * Delete a SKU.
+   * Retrieves the details of an existing SKU. Supply the unique SKU identifier from either a SKU
+   * creation request or from the product, and Stripe will return the corresponding SKU information.
    */
-  public Sku delete() throws StripeException {
-    return delete(null);
+  public static Sku retrieve(String id, RequestOptions options) throws StripeException {
+    return retrieve(id, (Map<String, Object>) null, options);
   }
 
   /**
-   * Delete a SKU.
+   * Retrieves the details of an existing SKU. Supply the unique SKU identifier from either a SKU
+   * creation request or from the product, and Stripe will return the corresponding SKU information.
    */
-  public Sku delete(RequestOptions options) throws StripeException {
-    return request(RequestMethod.DELETE, instanceUrl(Sku.class, this.id), null, Sku.class,
-        options);
+  public static Sku retrieve(String id, Map<String, Object> params, RequestOptions options)
+      throws StripeException {
+    String url = String.format("%s%s", Stripe.getApiBase(), String.format("/v1/skus/%s", id));
+    return request(ApiResource.RequestMethod.GET, url, params, Sku.class, options);
```

Comment:
5) Retrieve (standard): Static method to retrieve a resource with a given ID

Reply:
https://github.com/stripe/stripe-java/blob/2aa5db5c39dc6e91228507798083f373904cc8c0/src/main/java/com/stripe/model/Sku.java#L131-L139